app/src/main/java/com/babylon/wallet/android/presentation/transaction/fees/TransactionFeesDelegate.kt

```diff
@@ -66,8 +70,16 @@ class TransactionFeesDelegateImpl @Inject constructor(
     private var searchFeePayersJob: Job? = null
 
     suspend fun resolveFees() {
+        // The fees are not required for pre-authorized transactions
+        if (data.value.request.transactionType is TransactionType.PreAuthorized) {
+            return
+        }
+
         _state.update { it.copy(fees = TransactionReviewViewModel.State.Fees(isNetworkFeeLoading = true)) }
-        val executionSummary = data.value.transactionToReviewData.transactionToReview.executionSummary
+        val transactionToReviewData = data.filter { it.isTransactionToReviewDataReady }
```

Comment:
This fixes the crash you reported @micbakos-rdx, it happens because the analysis failed but it still was unnecessarily trying to resolve the fees